src/components/SignUp/Step0/index.tsx

```diff
@@ -0,0 +1,75 @@
+import { IChoice } from 'interfaces/auth';
+import { ChangeEvent, FC, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { setItem } from 'utils/localStorageFn';
+import ChoiceButton from '../ChoiceButton';
+import styles from './index.module.scss';
+
+interface iStep0 {
+  setStep: React.Dispatch<React.SetStateAction<string | null>>;
+}
+
+const choicesArray = [
+  { id: 0, value: 'I am a builder looking to build with projects 🧑‍💻' },
+  { id: 1, value: 'I have an organisation  🚀' },
+];
+
+const Step0: FC<iStep0> = ({ setStep }) => {
+  const [activeButton, setActiveButton] = useState<string | null>(null);
+  const navigate = useNavigate();
+
+  const handleNameChange = (e: ChangeEvent<HTMLInputElement>) => {
+    const { value } = e.target;
+    setItem('name', value);
+  };
+
+  const handleChoiceSelection = (choice: IChoice) => {
+    setActiveButton(`step${choice.id}`);
+  };
+
+  const handleSubmit = () => {
+    setStep(activeButton);
+    if (activeButton === 'step0') setStep('step3');
```

Comment:
I think it'll be better to have the value of `activeButton` as builder and organisation or something else. **step0** is getting mixed with form step.